.gitignore

```diff
@@ -9,4 +9,4 @@ HDL/*.rpt
 Software/**/.vscode
 Software/bin_amiga
 Software/bin_pi
-Software/bpls2gif/build
+Software/bpls2gif/build
```

Comment:
Could you add a new line character after this line? Now this line ends without a new line character.